Ginger/Ginger/Actions/ActionConversion/ConversionConfigurationWzardPage.xaml.cs

```diff
@@ -0,0 +1,165 @@
+            }
+        }
+
+        public ConversionConfigurationWzardPage()
+        {
+            InitializeComponent();
+        }
+
+        public void WizardEvent(WizardEventArgs WizardEventArgs)
+        {
+            switch (WizardEventArgs.EventType)
+            {
+                case EventType.Init:
+                    mWizard = (ActionsConversionWizard)WizardEventArgs.Wizard;
+                    break;
+                case EventType.Active:
+                    Init(WizardEventArgs);
+                    break;
+                case EventType.Finish:
+                    break;
+            }
+        }
+        
+        private void Init(WizardEventArgs WizardEventArgs)
+        {
+            cmbTargetApp.BindControl(mWizard.BusinessFlow.TargetApplications.Select(x => x.Name).ToList());
+            if ((cmbTargetApp != null) && (cmbTargetApp.Items.Count > 0))
+            {
+                cmbTargetApp.SelectedIndex = 0;
+            }
+            mWizard.ChkDefaultTargetApp = Convert.ToBoolean(chkDefaultTargetApp.IsChecked);
+
+            grdPane.RowDefinitions[2].Height = new GridLength(0);
+
+            LocateByPOMElementPage locateByPOMElementPage = new LocateByPOMElementPage(null, nameof(ActUIElement.ElementType), this, nameof(SelectedPom), true);
+            LocateValueEditFrame.Content = locateByPOMElementPage;
+        }
+
+        private void chkDefaultTargetApp_Checked(object sender, RoutedEventArgs e)
+        {
+            if ((cmbTargetApp != null) && (cmbTargetApp.Items.Count > 0))
+            {
+                mWizard.ChkDefaultTargetApp = Convert.ToBoolean(chkDefaultTargetApp.IsChecked);
+                cmbTargetApp.IsEnabled = true;
+                btnRefresh.IsEnabled = true;
+            }
+        }
+
+        private void chkDefaultTargetApp_Unchecked(object sender, RoutedEventArgs e)
+        {
+            if ((cmbTargetApp != null) && (cmbTargetApp.Items.Count > 0))
+            {
+                mWizard.ChkDefaultTargetApp = Convert.ToBoolean(chkDefaultTargetApp.IsChecked);
+                cmbTargetApp.IsEnabled = false;
+                btnRefresh.IsEnabled = false;
+            }
+        }
+
+        private void btnRefresh_Click(object sender, RoutedEventArgs e)
+        {
+            cmbTargetApp.BindControl(mWizard.BusinessFlow.TargetApplications.Select(x => x.Name).ToList());
+            if ((cmbTargetApp != null) && (cmbTargetApp.Items.Count > 0))
+            {
+                cmbTargetApp.SelectedIndex = 0;
+            }
+        }
+
+        private void CmbTargetApp_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            if(cmbTargetApp.SelectedIndex > -1)
+            {
+                mWizard.CmbTargetApp = Convert.ToString(cmbTargetApp.SelectedValue);
+            }
+        }
+
+        private void RadNewActivity_Checked(object sender, RoutedEventArgs e)
+        {
+            if (mWizard != null)
+            {
```

Comment:
Done, tried to do all things by binding to the properties in wizard page, but since i need to hide the grid row so the datacontext was required to be the conversionpage, i required to add the new properties in the conversion page, i tried to hide the grid row using binding but somehow it was not working.